presto-common/src/main/java/com/facebook/presto/common/type/LongDecimalType.java

```diff
@@ -29,7 +29,7 @@
 import static com.facebook.presto.common.type.UnscaledDecimal128Arithmetic.compare;
 import static io.airlift.slice.SizeOf.SIZE_OF_LONG;
 
-final class LongDecimalType
+public final class LongDecimalType
```

Comment:
Why is this needed?

Reply:
It's not anymore, I initially added it to check which decimal type we had in test metadata creation. Good catch